### PR TITLE
clean up project

### LINKED ISSUE #35: clean up project and system wide upgrade
remove unused routes and systems, including ec2 instance and firebase cloud functions

new data transfer route should be
```
hardware -> lambda (act as buffer) -> phoenix -> sls client -> users
```

if there is a need for data mining in the future, 2 things must be done:
1. make sure that the hardware is stable
2. build a dedicated server just to store loads of washer data entries

### PR DESCRIPTION
**Pull request by 98sean98**
closes #35 

deleted:
ec2 instance

deployed:
lambda function

not deployed:
sls website

### CODE REVIEW

**Comment by utkarsh867** [comment]
Gonna look into it soon

File: lambda/server.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+env.config();
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+const PORT = 4400;
+
+app.post('/', async (req, res) => {
+  const testUrl = 'http://localhost:4000/graphql'; // phoenix development server
```
Comment (utkarsh867) [inline lambda/server.js:15]: How did you test this?

**Review by utkarsh867 — COMMENTED** [review]
Looks great! 

I am curious to know if it has been tested, and it works? You can go ahead and merge if it is the case.

**Comment by utkarsh867** [comment]
As I remember, the node_modules in the lambda folder exist for a reason.


Please do make sure that you tested all changes before you break a very flaky system as it is.

File: lambda/server.js (hunk 1 above)
Comment (98sean98) [reply lambda/server.js:15]: spin up a local test server, and hit it with Postman
tried my best to simulate how the lambda function would be invoked with payload `Content-Type` = `application/json`, and identical payload data signature

**Comment by 98sean98** [comment]
> As I remember, the node_modules in the lambda folder exist for a reason.
> 
> Please do make sure that you tested all changes before you break a very flaky system as it is.

Yes. This is correct. I'll remove `node_modules` from `lambda/.gitignore`

**Comment by 98sean98** [comment]
this requires some rework and testing to ensure the `lambda` function can pass new `cors` settings in phoenix

**Pull request closed without merging**